content/References/IPv6.md

```diff
@@ -0,0 +1,239 @@
+
+  Assign either a global unicast address for publicly-accessible systems or use unique local addresses for private internal-only interfaces.
+
+* What type of auto-configuration should you use?
+
+  IPv6 supports stateful and stateless auto-configuration (SLAAC).
+
+  Stateful auto-configuration requires some human intervention. It needs a DHCPv6 server for the installation and administration of nodes.
+  The DHCPv6 server maintains state information so the server knows how long each address is in use and when it might be available for reassignment.
+
+  Stateless auto-configuration (SLAAC) is suitable for small organizations and individuals.
+  SLAAC can automatically configure IPv6 host parameters on an IPv6 host without needing an IT admin to manually configure or use a DHCPv6 server.
+  Each host determines its addresses from the contents of received router advertisements and using the IEE EUI-64 standard to define the network ID portion of the address, to assume uniqueness of the host address on the link.
+  For more information see [IPv6 Stateless Address Auto-configuration (SLAAC)](https://www.networkacademy.io/ccna/ipv6/stateless-address-autoconfiguration-slaac).
+  
+  <!-- commenting out until we can verify IPv6 on an HA system. At present HA fails using assigned network addresses at failover.
+  {{< enterprise >}}
+  SLAAC can be enabled on any pubic or private subnet, but should not be used with an HA supported system because they use an virtual IP address.
+  We recommend configuring HA systems on unique VLANs without DHCP or SLAAC support enabled.
+  {{< /enterprise >}} 
+  -->
+
+* Do you need to route using both IPv6 and IPv4 addresses?
+
+  If you want to deploy an all IPv6 network or all IPv4 network, you need to address the requirements of each and assess your network infrastructure.
+  There are several ways to implement IPv6 alone or to dual-stack both IP protocols in the same network.
+
+  Because IPv6 is not widely implemented, and neither IP protocol can directly communicate without configuring your network to accommodate both IPv4 and IPv6 nodes in the same network, you need to choose how you want to implement IPv6.
+  At present, IT department options are to dual-stack the protocols, create an IPv6 tunnel to carry IPv6 packets over IPv4 infrastructure, or use a gateway to translate between the two address structures.
+
+  For the network infrastructure, begin by enabling both TCP/IP protocol stacks on the wide area network (WAN) core routers, the perimeter routers and firewalls, followed by the data-center routers/desktop access routers, and finally the end user equipment and devices.
+
+* Can you dual stack IPv4 and IPv6 in SCALE?
+  
+  Yes! You can configure SCALE with a dual-stack of both if your system has either two network interface cards or a NIC with two ports.
+  After wiring both ports or each NIC, configure the SCALE UI with the assigned addresses, and then create a VLAN bridge.
```

Comment:
A VLAN bridge does not have to be part of an IPv6 network setup.

Reply:
This content is removed for now